lib/msf/core/handler/reverse_http.rb

```diff
@@ -239,8 +239,7 @@ def setup_handler
     lookup_proxy_settings
 
     if datastore['IgnoreUnknownPayloads']
-      payload_count = framework.db.payloads({workspace: framework.db.workspace}).length
-      print_status("Handler is ignoring unknown payloads, there are #{payload_count} UUIDs whitelisted")
+      print_status("Handler is ignoring unknown payloads")
```

Comment:
The count offered had some value, it may be work simply removing workspace scope there as knowing that all payloads are being rejected if none are in the db would be a helpful flag to the user.

Reply:
I didn't see a way to lookup all payloads in the database, so I dropped it. It seems like payloads can only be queries by id or uuid. Any tips on how to get all given the available functions?

Reply:
For the purpose of the bugfix, I think we might skip this for now.